### ISSUE TITLE
Using the dialogue with commands

### ISSUE THREAD

**Issue by Devillion** [ezeiger92/QuestWorld2#73]
This doesn't always work with commands.

Some commands, when you type them in don't get saved to the dialogue and the command runs when you hit enter.

I'm assuming it's supposed to block whatever commands you type from running until you get out of that mode with exit()

**Comment by ezeiger92** [comment]
Looks like I support having commands in dialogue, but I don’t support actually adding them. Woops.

**Comment by Fharen** [comment]
This happened to me before, I simply edited one of the config files so that questworld will not add color to the dialog automatically.

**Comment by ezeiger92** [comment]
@Fharen Not the same issue, sadly. The dialogue prefixes no longer saved in files and are only added to lines of text. When I removed the old input system for dialogue, I forgot to add a listener for commands. That's all lol.

@Devillion I've got too much going on at the moment to test and put out a proper build. I might have some time over the weekend or by Tuesday.
For now, if you manually edit the dialogue files they will let you run any commands you put in. I think /qe reload quests will load changes while the server is running.
